### ISSUE TITLE
Add plugins for PHP, CSS support

### ISSUE THREAD

**Issue by dstrunk** [dstrunk/dotfiles#3]
These plugins need to be added for PHP and CSS support

``` vim
Plugin 'stanangeloff/php.vim'
Plugin 'joonty/vim-phpqa'
Plugin 'Ioannis-Kapoulas/vim-autoprefixer'
```

also note that per [this issue](https://github.com/Ioannis-Kapoulas/vim-autoprefixer/issues/2) the autoprefixer cli tool is deprecated in favor of `postcss --use autoprefixer`. An easy fix for this is to add this alias to bash:

``` sh
alias autoprefixer="postcss --use autoprefixer"
```

**Comment by Ioannis-Kapoulas** [comment]
vim-autoprefixer now works with the new postcss-cli. 
The instructions have been updated.

**Comment by dstrunk** [comment]
closed in 6fd7fcc
